lib/omnibus/licensing.rb

```diff
@@ -95,6 +116,8 @@ def prepare
       FileUtils.rm_rf(output_dir)
       FileUtils.mkdir_p(output_dir)
       FileUtils.touch(output_dir_gitkeep_file)
+      FileUtils.rm_rf(cache_dir)
+      FileUtils.mkdir_p(cache_dir)
```

Comment:
👍 Will add that. I think it dropped during the merge 😄 

Also yes. I think we should merge #704 first and then bring this in. We will need to also bump the version with this and make sure omnibus-software is pinned to the version we bump to since we will make use of `skip_transitive_dependency_licensing` there.